snapshot/cli/cli.go

```diff
@@ -229,9 +231,17 @@ func (c *createGitBundleCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 		}
 		tempDir = td.Dir
 
+		// Get the length of this revList for the sha we will compute for file name
```

Comment:
I don't quite follow why this is the chosen solution. Can we get a sha based on just the revList? Or can you explain a little more.